Objective-C/CBLConsoleLogger.m

```diff
@@ -32,6 +32,7 @@ - (instancetype) initWithLogLevel: (CBLLogLevel)level {
     if (self) {
         _level = level;
         _domains = kCBLLogDomainAll;
+        _sysID = [[NSBundle mainBundle] bundleIdentifier];
```

Comment:
I'm not sure if this is right as it think it will return the bundle identifier of the app not the framework. What is the value when you run the test? If this returns bundle id of the app, you may try to get the bundle id from one of the classes such as CBLConsoleLogger itself.

Reply:
you are right, will fix